lib/resty/radixtree.lua

```diff
@@ -130,6 +130,7 @@ end
 
 
 local _M = { _VERSION = 1.7 }
+_M.symbols = radix
```

Comment:
style: use `_symbols`. Remember to add comment to show it is exposed for test.

Reply:
ok